tools/deps/wingetopt/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+PROJECT(wingetopt)
+cmake_minimum_required(VERSION 2.8)
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src)
+add_library(wingetopt STATIC src/getopt.c src/getopt.h)
+
+install(FILES src/getopt.h DESTINATION include)
+install(TARGETS wingetopt LIBRARY DESTINATION lib ARCHIVE DESTINATION lib)
```

Comment:
Why do we need install steps here? This could be compiled as a static library and the executable that depend on it could link this library statically into their executable.

Reply:
fixed.